### PR TITLE
feat(frontend): added example frontend

### PR DESCRIPTION
**Pull request by bpowell**

### CODE REVIEW

File: frontend/rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import builtins from 'rollup-plugin-node-builtins';
+import babel from 'rollup-plugin-babel';
+import typescript from 'rollup-plugin-typescript2'
+import commonjs from 'rollup-plugin-commonjs'
+// import external from 'rollup-plugin-peer-deps-external'
```
Comment (link108) [inline frontend/rollup.config.js]: remove commented lines

File: frontend/src/test.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import ExampleComponent from './'
+
+describe('ExampleComponent', () => {
```
Comment (link108) [inline frontend/src/test.ts]: seems like a fake test, may want to remove it or add a test that is valid.

**Comment by link108** [comment]
I'd like to see some more comments around how this is used/how it works, since this is going to be an example repo that other devs will base their plugins on

**Comment by bpowell** [comment]
@link108 updated with comments, let me know if that makes sense or stuff needs to be explained more

**Comment by link108** [comment]
@bpowell Imagine if someone wants to make a plugin for Spinnaker and they haven't checked out that code at all, eg Deck and Orca. Are they able to look at this repo and understand what is going on? I think the comments you left are more around what is going on instead of the why or explaining what is going on, so you're closer, but not quite all the way there. Happy to spend 15-20 min on this with you later today on this

**Pull request merged**